app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -42,18 +47,38 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               label="Title"
               name="title"
             />
-            <Textarea
+            <Input
               required
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+            />
+            <Select
+              defaultMenuIsClose
```

Comment:
Please share the docs regarding this prop. Also, what happens if we don't provide this prop?

Reply:
Oops! I've no idea where did I find this prop. 
I realize that it's not a valid prop and it makes no effect.